spec/spec_helper.cr

```diff
@@ -1,4 +1,8 @@
 require "spec"
 require "colorize"
 
-ENV["CRYSTAL_ENV"] = "TEST"
\ No newline at end of file
+ENV["CRYSTAL_ENV"] = "TEST"
+
+# This forces use of cluster-tools manifest in the repo's tools dir when running specs.
+ENV["USE_MANIFEST_FOR_SPECS"] = "yes"
```

Comment:
can we make this a boolean? probably doesn't matter but there are only 2 states I can think of for this

Reply:
ENV only allows String keys and String values. So had to use some string there.